src/Modules/score.js

```diff
@@ -1,9 +1,15 @@
-const { v4: uuidv4 } = require('uuid');
+async function userScores(a, b) {
+  const BaseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/GrlP35gyONINxwNnhDtc/scores';
```

Comment:
- [ optional] you can use the `async` alongside an `await` expression to ensure the code runs synchronously.